packages/astro/src/@types/astro.ts

```diff
@@ -1448,9 +1489,9 @@ export interface AstroUserConfig {
 		 * @type {'shiki' | 'prism' | false}
 		 * @default `shiki`
 		 * @description
-		 * Which syntax highlighter to use, if any.
-		 * - `shiki` - use the [Shiki](https://shiki.style) highlighter
-		 * - `prism` - use the [Prism](https://prismjs.com/) highlighter
+		 * Which syntax highlighter to use for Markdown code blocks (/`/`/`), if any. This determines the CSS classes that Astro will apply to your Markdown code blocks.
```

Comment:
```suggestion
		 * Which syntax highlighter to use for Markdown code blocks (\`\`\`), if any. This determines the CSS classes that Astro will apply to your Markdown code blocks.
```

The backticks doesn't appear to be outputting correctly.